ipywidgets_bokeh/src/loader.ts

```diff
@@ -15,13 +15,13 @@ function get_cdn_url(moduleName: string, moduleVersion: string, cdn: string) {
   // if a '/' is present, like 'foo/bar', packageName is changed to 'foo', and path to 'bar'
   // We first find the first '/'
   let index = moduleName.indexOf("/")
-  if ((index != -1) && (moduleName[0] == "@")) {
+  if ((index !== -1) && (moduleName[0] === "@")) {
```

Comment:
> and fixes a few != and == where they should be !== and ===.

Does this actually fix anything? In this case we are comparing numbers to numbers and strings to strings, so there's no implicit behavior here. As pattern in bokehjs we use short form in general and long form when the distinction is required (e.g. where `null` and `undefined` have their own meaning, though in those cases we are migrating to explicit unique symbols, to avoid any confusion).

Reply:
`fix` is definitely the wrong word to have used. The linter I'm using was configured incorrectly, which I've updated.